src/javascript/app/pages/cashier/cashier.js

```diff
@@ -43,6 +43,19 @@ const Cashier = (() => {
         });
     };
 
+    const showP2PNote = async () => {
+        const is_agent = !(await BinarySocket.send({ p2p_agent_info: 1 })).error;
+        const has_buy = await checkP2PHasOffer('buy');
```

Comment:
IMO we should check is_agent first, so in case it's true we don't need to make extra calls